pages/1.11/storage/external-storage/index.md

```diff
@@ -148,3 +151,4 @@ If you scale your app down to 0 instances, the volume is detached from the agent
 [10]: https://github.com/emccode/dvdcli#extra-options
 [11]: https://rexray.readthedocs.io/en/v0.9.0/user-guide/schedulers/#docker-containerizer-with-marathon
 [12]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-attaching-volume.html
+[13]: https://rexray.readthedocs.io/en/stable/user-guide/storage-providers/aws/#configuration-notes
```

Comment:
the other doc links point to RR 0.9 .. should this as well?

Reply:
Great point

Reply:
oh that page doesn't exist in 0.9 :'(

Reply:
> REX-Ray by default will fail after 13 EBS volumes are attached. Later versions of REX-Ray have a parameter to overcome, useLargeDeviceRange in the the REX-Ray 0.11.0 EBS docs page. DC/OS v1.11.0 bundles REX-Ray 0.9.0.

Reply:
?